changesLog.js

```diff
@@ -1,34 +1,75 @@
+const octokit = require('./githubAPI/octokit');
 const commander = require('commander');
-const { getIssuesByMilestone } = require('./changesLogGenerator');
+const fs = require('fs');
+const { fetchIssuesInMilestone, generateChangesLog } = require('./helpers');
 
-function commaSeparatedList(value, dummyPrevious) {
+function commaSeparatedList(value) {
   return value.split(',');
 }
 
+function checkValidStates(value) {
+  return (value === 'closed') || (value === 'open') || (value === 'all') ? value : undefined;
+}
+
 commander
   .version('1.0.0', '-v, --version')
   .usage('[OPTIONS]...')
-  .requiredOption('-m, --milestone <name>', 'must filter issue from milestone')
-  .option('-d, --duplicate <allow>', 'Should allow duplicated issues (Default: true)')
-  .option('-f, --filters <items>', 'comma separeted labels to filter issues', commaSeparatedList)
-  .option('-g, --groups <items>', 'comma separated labels to group by issues', commaSeparatedList)
+  .requiredOption('-m, --milestone <number>', 'must filter issue from milestone')
+  .option('-c, --customer <customer>, will filter the issues related to a customer if this option is received')
+  .option('-i, --includeAll <flag>', 'will include a section with issues done for all clients (Default: false)')
+  .option('-s, --state <state>', 'should use issues with state: closed, open, all (Default: closed)', checkValidStates)
+  .option('-f, --filters <labels>', 'comma separated labels to filter and group by issues', commaSeparatedList)
+  .option('-d, --duplicate <allow>', 'should allow duplicated issues in different groups (Default: true)')
```

Comment:
I don't agree. I feel that this way, since that's the only place this is used it looks more easy to read like this. Please, let me know if you are strong about this and we can discuss the idea.